cs/commandstation/cm_test_helper.hxx

```diff
@@ -19,6 +19,18 @@ extern Pool* const g_incoming_datagram_allocator = init_main_buffer_pool();
 
 namespace commandstation {
 
+class DccPacketSink : public dcc::PacketFlowInterface {
```

Comment:
Comment as to what this object does? Comment its single method?

Reply:
done